.github/workflows/main.yml

```diff
@@ -76,24 +80,32 @@ jobs:
           python: '3.10'
           allow_failure: false
 
-        - name: py39-dj32-postgres-xdist-coverage
+        - name: py39-dj41-mysql_innodb-coverage
```

Comment:
did a little rearranging of order so these are always in descending order by python version, then by django version - they were previously inconsistent